docs: update oci documentation and examples.

karwandevishal opened the pull request:
Adding Oracle Generative AI as one of the providers for langchain.
Updated the old examples in the documentation with the new working examples.

Commit:
added oci as a provider and updated test examples

added in oci in dropdown

updated docs

karwandevishal committed:
Merge branch 'master' into release/oci_doc_update

Commit:
lint changes

Commit:
doc link updated

YouNeedCryDear commented on libs/packages.yml:
```diff
@@ -733,4 +733,8 @@ packages:
 - name: langchain-scrapeless
   repo: scrapeless-ai/langchain-scrapeless
   path: .
+- name: oracleai
```
Is this the correct name?

karwandevishal replied:
i think this will be displayed name_title: Oracle Generative AI. Since document is available by oracleai in providers folder kept the same name. One of the check was failing.

mdrxy committed:
release(text-splitters): 0.3.11 (#32770)

Fixes #32747

SpaCy integration test fixture was trying to use pip to download the
SpaCy language model (`en_core_web_sm`), but uv-managed environments
don't include pip by default. Fail test if not installed as opposed to
downloading.

added caution for older text generation messages

updated chat documentation

karwandevishal committed:
Merge branch 'master' into release/oci_doc_update

Commit:
updated embedding example

lint changes

updated embedding example

lint changes

reverted old changes

fixed markdown

added caution

merge

Commit:
renamed file, added oci in chat and embedding examples

Commit:
updated caution

Commit:
fixed type

Commit:
updated name and links for embeddings

piyushg reviewed (approved):
Approved

Commit:
added repo link and missed link for ChatOCIGenAI

Commit:
updated link to oracle repo

Commit:
updated link to oracle repo in embeddings

karwandevishal committed:
Merge branch 'master' into release/oci_doc_update

karwandevishal committed:
Merge branch 'master' into release/oci_doc_update

mdrxy committed:
Merge branch 'master' into release/oci_doc_update
